website/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Dynamoose | Quick-start</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Raleway:400,500"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto+Mono"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/styles/default.min.css"
+    />
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/highlight.min.js"></script>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" type="text/css" media="screen" href="index.css" />
+
+    <script src="index.js"></script>
+    <script
+      src="https://embed.runkit.com"
+      data-element-id="my-element"
+    ></script>
+  </head>
+
+  <body>
+    <div id="header">
+      <p id="logo">Dynamoose</p>
+      <ul id="nav">
+        <li><a href="index.html">Quick-start</a></li>
+        <li><a href="examples.html">Examples</a></li>
+        <li><a href="https://hweeks.github.io/dynamoose/">API Docs</a></li>
+      </ul>
+    </div>
+
+    <div id="hero">
+      <div id="download">
+        <p id="hero-title">Dynamoose</p>
+        <p id="hero-subtitle">
+          A modelling tool for Amazon's <span>DynamoDB</span>.
+          <br />
+          Inspired by Mongoose.
+        </p>
+        <div id="code-box">
+          <div id="code-box-top-corner">
+            <i class="fas fa-circle"></i>
+            <i class="fas fa-circle"></i>
+            <i class="fas fa-circle"></i>
+          </div>
+          <div id="code-box-command">
+            <span id="prompt">$</span>
+            <p>
+              <span id="prompt">npm</span> install
+              <span id="npmname">dynamoose</span> --save
+            </p>
+          </div>
+        </div>
+      </div>
+      <button id="get-started">
+        Get started
+        <i style="padding: 0 0 0 10px;" class="fas fa-arrow-right"></i>
```

Comment:
We shouldn't mix inline and style-sheet based styling. inline takes is the most specific styling and cannot be overridden.